tests/test_llm_cost.py

```
@@ -0,0 +1,29 @@
+import os
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)